src/backend/distributed/citus--5.0-1--5.0-2.sql

```diff
@@ -0,0 +1,8 @@
+/* citus--5.0-1--5.0-2.sql */
+
+CREATE FUNCTION master_update_shard_statistics(relation_id regclass, shard_id bigint)
```

Comment:
This is pretty minor, but it seems like we don't really need the user to specify the relationId in this function. You could just look that up using LoadShardInterval.